docs/multiple-remotes.md

```diff
@@ -0,0 +1,34 @@
+## Pushing to multiple remotes
+
+As it stands, a Gitkube remote is just for deployment and should not be considered as a storage medium like GitHub.
+If however one would like for git push to both deploy to a kubernetes cluster as well as store code for the long haul in GitHub,
+this can be achieved.
+
+One simple way to achieve this is to begin with an origin remote pointing to Github and then add the Gitkube repository as a push
+url for the origin remote. The following example presumes that you already have two remotes: one pointing to Github and the other
+to Gitkube.
+
+```
+$ git remote -v
+origin  git@github.com/<github-owner>:<github-repo>.git (fetch)
+origin  git@github.com/<github-owner>:<github-repo>.git (push)
+myremote	ssh://<namespace>-<remote-name>@<any-node-ip>:<node-port>/~/git/<namespace>-<remote-name> (fetch)
```

Comment:
The `myremote` url is specific to gitkubed which is exposed as NodePort. Can you just put ` <gitkube-remote-url>` instead of the templated string.